bottlerocket-settings-models/settings-models/src/kubernetes/mod.rs

```diff
@@ -24,6 +24,10 @@ use bottlerocket_modeled_types::{
     TopologyManagerScope, Url, ValidBase64, ValidLinuxHostname,
 };
 
+// Feature flags
+#[cfg(feature = "nvidia-device-plugin")]
```

Comment:
We also need to set the variable to false in the absence of the feature.

Reply:
Ah, you'll also need the variable to be `pub` to be accessible from the extension side.

Reply:
I added that.